### ISSUE TITLE
How to Test in every epoch with Brainscript

### ISSUE THREAD

**Issue by Eomtaesan** [Microsoft/CNTK#2509]
Hi, I'm using brainscript.
I make and run a simple dnn.
CNTK produce model files in models folder. (simple.dnn.0, simple.dnn.1, ... , simple.dnn.99, ... simple.dnn)
I can see decreasing error in log file, but the error is about training set.
To see error about test data set, I have to edit simple.cntk file
( command = Train --> command = Test, modelpth = "Models/simple.dnn.0" --> modelpth = "Models/simple.dnn.1" )
and I run the edited cntk file.

But that is so stupid. I think there is a way to easy and simple.
How to see the error about test data set in every epoch?

**Comment by eldakms** [comment]
Please use cvReader in your Train section. For samples have a look here:
https://github.com/Microsoft/CNTK/blob/master/Examples/Image/Classification/ResNet/BrainScript/ResNet101_ImageNet1K.cntk
Thanks!

**Comment by Eomtaesan** [comment]
@eldakms 
I see the site.
I wonder this if I specify "cvReader", then the error is about reader's file or cvReader's file?

**Comment by eldakms** [comment]
In cvReader you give a test set file, so there will be a separate line for evaluation on this set , something like: 
Finished Epoch[blabla]: **[Validate]** ...

**Comment by Eomtaesan** [comment]
@eldakms 
wow good, I tried, and see the message.
Thank you so much quick response, and you correctly solve my question.
Thank you.
